community/kernel/src/main/java/org/neo4j/kernel/configuration/ssl/SslPolicyConfig.java

```diff
@@ -44,7 +45,8 @@
 @Group( "dbms.ssl.policy" )
 public class SslPolicyConfig
 {
-    private static final String TLS_VERSION_DEFAULTS = join( ",", new String[]{"TLSv1.2"} );
+    public static final List<String> TLS_VERSION_DEFAULTS = asList( "TLSv1.2" );
+    public static final List<String> CIPHER_SUITES_DEFAULTS = null;
```

Comment:
please use the empty list here instead, you digging a grave for your own joinList that need to check if the value is null; plus making future code not null-safe

Reply:
It has to be null because of the two lines added in the `SslSocketConnectorFactory.java`.
If it is null, then it will default to use jetty's cipher filter, otherwise if it an empty list (which means a user wrote this config on purpose), then it will overwrite the jetty's default filter with `sslContextFactory.setExcludeCipherSuites();`.